src/watchmaker/config/status/__init__.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+"""Status Config module."""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals, with_statement)
+
+import logging
+
+STATUS = {
+    "RUNNING": "Running",
+    "COMPLETE": "Completed",
+    "ERROR": "Error",
+}
+
+
+def is_valid_status_config(config):
+    """Validate config."""
+    is_valid = True
+    if config and "targets" in config:
+        for target in config["targets"]:
+            if "key" not in target or not target["key"]:
+                is_valid = False
+                logging.error("Status target is missing key or value")
+            if "target_type" not in target or not target["target_type"]:
+                is_valid = False
+                logging.error("Status target is missing target_type or value")
+            if "required" in target:
+                if not isinstance(target["required"], bool):
+                    is_valid = False
+                    logging.error("Status target required value is not a bool")
+
+    return is_valid
+
+
+def get_status(status_key):
+    """Get formatted status message for status key provided."""
+    status = STATUS.get(status_key, None)
+    return status if status else status_key
```

Comment:
I was allowing an option for custom messages or use pre-canned messages.

Reply:
Can you elaborate how the custom messages work? This is just returning the key... but the key to what?

Reply:
If the key is a valid key in the STATUS dict
`STATUS = {
    "RUNNING": "Running",
    "COMPLETE": "Completed",
    "ERROR": "Error",
}`

it would return the value for that key as the status, otherwise it just returns what was pass in as the status.   
Essentially, the status is either a value from the map or what was passed in.  That would be addressed in the comments when I go through.  This was refactored a few times so I was holding off on that to document.

Is this necessary, no, but I thought what if we wanted a more elaborate message for one tag resource spot in the code, this would cover that use case without having another tag_resource method.

Reply:
Ok, so sometimes the `status_key` is not a key, but the custom message? Like so?

```
get_status("RUNNING")
# "Running"

get_status("My custom status message here")
# "My custom status message here"
```